etl/calculate-and-feed-indicators.ts

```
@@ -0,0 +1,308 @@
+        | { MACD?: number; signal?: number; adx?: number; k?: number; d?: number; upper?: number; lower?: number }
+        | undefined
+      )[],
+      index: number,
+      path?: string
+    ) => {
+      const val = arr[index]
+      if (val === undefined || val === null) return 0
+      if (typeof val === "number") return val
+      if (typeof val === "object" && path) {
+        const keys = path.split(".")
+        let current: any = val
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)